posts/azure-seen-by-developers.md

```diff
@@ -0,0 +1,190 @@
+
+### Containers
+
+![Figure 20 : the storage containers](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_20.png)
+
+This view lists all the containers created in the storage. It allows to create new ones, to modify them after creation and to delete them. Let's simulate the creation of a new container.
+
+![Figure 21 : creation of a new Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_21.png)
+
+This view is the interface for creating a new container. It is very simple, requiring only the name of the new container and the access level. When you click on the drop-down list, the three options shown in Figure 22 are displayed.
+
+![Figure 22 : editing the access level of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_22.png)
+
+The three levels of access to a container are _Private_, _Blob_ and _Container_. Configured on Private, a token is required to access the resources of the container. Configured on Blob, it is possible to access the resources hosted in the storage without an access token. Finally, configured on Container, it is possible to access the contents of the containers without access token.
+
+![Figure 23 : editing the access level of a Container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_23.png)
+
+It is possible to change the accessibility level of a container even after it has been created by right-clicking on it, which can be very useful.
+
+Finally, enter the container to view its contents.
+
+![Figure 24 : content of a Blob container](https://github.com/aversaveau/blog-usa/blob/azureSeenByDevs/images/2021/03/azure_seen_by_devs_24.png)
+
+This view lists all the blobs contained in a container. A container can contain three types of blobs namely _Block blobs_, _Append blobs_ and _Page blobs_.
+
+_Block blobs_ are designed to store text or binary files, and to load large files efficiently. _Append blobs_ are designed for append operations such as logging. Finally, _Page blobs_ are made up of pages of up to 512 bytes that can reach a total size of 8 TB. They are designed for frequent random read/write operations. In the context of our project, we only use _Block blobs_.
+
+Thus, the view of a container allows to visualize the blobs contained in it as well as the public URL at which they are accessible, but also to upload new ones, to delete them and even to create access tokens specific to these blobs.
```

Comment:
The view of a container will now allow you to visualize the blobs contained in it, as well as the public URL at which they are accessible, but also to upload new ones, delete them, and even to create access tokens specific to these blobs.